### ISSUE TITLE
Turn off line-wrapping for code blocks please 😌

### ISSUE THREAD

**Issue by GraphicOwls** [notaapp/issues#246]
It would be great if we had an option to turn off line-wrapping for code blocks. If we had a quick copy button too, it would be a nice plus.

**Comment by astoilkov** [comment]
I agree. It would be an enhancement if code blocks didn't have to wrap.

For copying, have you tried ⌘L (select line) and then ⌘E (expand selection)? This will quickly select the contents of a code block. [Expand Selection – Nota Docs](https://docs.nota.md/expand-selection.html#:~:text=Another%20quick%20approach%20is%20to%20press%20%E2%8C%98L%20(expand%20selection%20to%20line)%20and%20then%20press%20%E2%8C%98E%20(expand%20selection))
